### ISSUE TITLE
"Interface -> Editor" and "Interface -> Editors"

### ISSUE THREAD

**Issue by aaronfranke** [godotengine/godot#25363]
**Godot version:** 3.1 beta 2 (applies with or without Mono, and in 3.0.6)

**OS/device including version:** Xubuntu 18.04 LTS 64-bit (should apply on any platform)

**Issue description:** There are several problems with these menus. Here are what they look like:

![e](https://user-images.githubusercontent.com/1646875/51797540-926ddb80-21d3-11e9-947e-46fdc6368043.png)

![d](https://user-images.githubusercontent.com/1646875/51797539-900b8180-21d3-11e9-963f-2715db9638b8.png)

I put check boxes before each item to track if they have been fixed.

- [ ] First of all, the "Editors" menu should probably be merged together with the "Editor" section. I see no obvious reason why these two settings are split into their own section, and the name "Editors" is far too similar to "Editor" so it could lead to confusion.

- [ ] Secondly, the Font entries from "Editor" should be moved to "Theme". It makes no sense to have font selection and size in "Editor" while font color is in "Theme".

- [ ] Thirdly, I noticed that a lot of these entries simply aren't sorted. It would be much easier to find settings if they were sorted in alphabetical order (unless there is a good reason to group things together). Instead of this:

![f](https://user-images.githubusercontent.com/1646875/51797622-e75e2180-21d4-11e9-8eb1-3f9ddfe80737.png)

How about this:

![g](https://user-images.githubusercontent.com/1646875/51797623-eaf1a880-21d4-11e9-9e51-d36ca846f589.png)

Pretty much every menu in Godot could be re-sorted. Though, this depends on #32144.

**Comment by EricEzaM** [comment]
Related/superseded by https://github.com/godotengine/godot-proposals/issues/1913?

**Comment by aaronfranke** [comment]
@EricEzaM Yeah, there is a lot of overlap, so we can track it all in your proposal.
